.gitignore

```diff
@@ -1,3 +1,4 @@
 dist
 .DS_STore
-node_modules
\ No newline at end of file
+node_modules
+.prettierrc.yml
```

Comment:
What tool does use this file?

Reply:
@hanzei The Prettier VS Code extension uses it: https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

Reply:
In that case I think it should be place in the global `.gitignore` and not the a repository spefic one.